src/main/java/org/apache/flink/benchmark/ContinuousFileReaderOperatorBenchmark.java

```diff
@@ -45,6 +51,7 @@
 
     private static final TimestampedFileInputSplit SPLIT = new TimestampedFileInputSplit(0, 0, new Path("."), 0, 0, new String[]{});
     private static final String LINE = Strings.repeat('0', 10);
+    private static OneShotLatch TARGET_COUNT_REACHED_LATCH = new OneShotLatch();
```

Comment:
Could you copy the explanation from the commit message into a comment here, describing the intention behind this latch?